index.html

```diff
@@ -4545,7 +4545,20 @@ <h2>
                 <li>Set <var>request</var>.<a>[[\state]]</a> to
                 "<a>closed</a>".
                 </li>
-                <li>Reject the promise
+                <li>If <var>request</var><a>[[\response]]</a> is not null,
+                then:
+                  <ol>
+                    <li>Let <var>response</var> be
+                    <var>request</var><a>[[\response]]</a>.
+                    </li>
+                    <li>Set <var>response</var><a>[[\complete]]</a> to true.
+                    </li>
+                    <li>Reject <var>response</var><a>[[\retryPromise]]</a> with
+                    <var>exception</var>.
+                    </li>
+                  </ol>
+                </li>
+                <li>Otherwise, reject
                 <var>request</var>.<a>[[\acceptPromise]]</a> with
```

Comment:
The note at the bottom seems like it needs updating to mention the retryPromise/retry() in addition to the acceptPromise/show().

Reply:
Made a few clarifications throughout the spec about this.